### PR TITLE
Add nostr login

### LINKED ISSUE #334: Nostr login
It's pretty trivial, but I keep forgetting to do this.

### PR DESCRIPTION
**Pull request by ekzyis**
https://github.com/stackernews/stacker.news/assets/27162016/331e1707-e99f-4fa5-9c31-56eb2d1210f9

Close #334 

TODO:
- [x] Use CSS for white icon
- [x] Add `Link Nostr`

### CODE REVIEW

File: components/nostr-auth.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      <div className='text-muted pb-4'>{details}</div>
+    </>
+  )
+}
+
+function NostrExplainer ({ text }) {
+  return (
+    <>
+      <div className='fw-bold text-muted pb-1'>error: nostr extension not found</div>
+      <div className='text-muted pb-4'>Nostr extensions are the safest way to use your nostr identity in Outer Space.</div>
```
Comment (ekzyis) [inline components/nostr-auth.js:25]: whoops, copy paste error:

> safest way to use your nostr identity in Outer Space

**Comment by huumn** [comment]
Looks good. But if you're using white text you should use white icons.

**Comment by ekzyis** [comment]
Oh, I forgot to add `Link Nostr` in settings

**Comment by huumn** [comment]
The logo shouldn't need to be statically colored. 

Look at the way we use our other svgs. We color them with CSS.

**Pull request merged**